azconfig/client/pom.xml

```diff
@@ -12,55 +12,23 @@
         <version>1.0.0-SNAPSHOT</version>
         <relativePath>../../pom.client.xml</relativePath>
     </parent>
+
+    <groupId>com.microsoft.azure</groupId>
     <artifactId>azconfig</artifactId>
```

Comment:
I've started an email thread with you to discuss the artifactId naming - let's see where that thread goes...but at present azconfig is not a good artifactId.

Reply:
I agree. 🤔